.github/workflows/auto-comment.yml

```diff
@@ -10,7 +10,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+
       - name: Process Label Action
         uses: hramos/respond-to-issue-based-on-label@v2
         with:
           repo-token: ${{ secrets.GITHUB_TOKEN }}
+          configuration-path: '.github/label-actions.yml'
```

Comment:
I noticed that the content for the `label-actions.yml` differs from what we have in the plugins/CICS client:

```
bug:
  comment: |
    Thank you for creating a bug report.
    We will investigate the bug and evaluate its impact on the product.
    If you haven't already, please ensure you have provided steps to reproduce the bug and as much context as possible.
enhancement:
  comment: |
    Thank you for raising this enhancement request.
    The community has 90 days to vote on it.
    If the enhancement receives at least 5 upvotes, it is added to our development backlog.
    If it receives fewer votes, the issue is closed.
```

Do we want to update these for consistency or leave as-is? It's not a result of the changes on this PR, but just figured I'd mention it 🙂

Reply:
Updated to match the other repos for consistency, except we decided at standup to keep 10 upvotes instead of 5.